src/dotnet-getdocument/Commands/InvokeCommand.cs

```diff
@@ -89,15 +89,22 @@ protected override int Execute()
                         break;
 
                     case ".NETCoreApp":
-                        executable = "dotnet";
-                        toolsDirectory = Path.Combine(thisPath, "netcoreapp2.0");
-
                         if (targetFramework.Version < new Version(2, 0))
                         {
                             throw new CommandException(
                                 Resources.FormatNETCoreApp1Project(project.Name, targetFramework.Version));
                         }
 
+                        var targetFrameworkPath = project.TargetFramework;
+                        if (targetFramework.Version > new Version(2, 2))
```

Comment:
Should this throw if the tfm is something the tool is newer than the most recently supported version?

Reply:
Depending on the newer version, it _might_ be compatible. And, the error will be more clear eg. a complaint about loading a specific assembly if we let the loader deal with it.